service/src/test/kotlin/no/nav/su/se/bakover/service/utbetaling/GjenopptaUtbetalingerServiceTest.kt

```diff
@@ -32,107 +28,41 @@ import no.nav.su.se.bakover.service.argThat
 import no.nav.su.se.bakover.service.fixedClock
```

Comment:
fjern?

Reply:
Sparer den til noen andre.. holder vel for denne pr'en nå.